src/helpers/buildHyperlinkHelper.ts

```diff
@@ -1,22 +1,29 @@
-import type { SelectedGeographyData } from "../types"
+import type { SelectedGeographyData } from "../types";
+import { areAllUndefined } from "../util/genUtil";
 
-export const buildHyperlink = (args: {topic?: string, variable?: string, classification?: string, category?: string, selectedGeography?: SelectedGeographyData }) => {
-    let url = "/"
-    if (args.topic){
-        if (args.category && !args.classification){
-            args.classification = "default"
-        }
-        const paramsArr = [args.topic, args.variable, args.classification, args.category]
-        url = "/2021"
-        paramsArr.forEach((param) => {
-            if (param){
-                url = `${url}/${param}`
-            }
-        })
+export const buildHyperlink = (args: {
+  topic?: string;
```

Comment:
what does the ? here do? Is this an optional arg?

Reply:
yes exactly, it's how you specify an optional argument in typescript

Reply:
wicked, thanks!